pkg/cmd/errors.go

```diff
@@ -0,0 +1,24 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/confluentinc/cli/v3/pkg/color"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+)
+
+func CheckErr(enableColor bool, msg interface{}) {
```

Comment:
This might be put together in pkg errors?

Reply:
I put this function in this package since it (and other functions in this package) mirrors a function inside the `cobra` package.